chef-server-stats/folsom-stats

```diff
@@ -7,12 +7,15 @@
 fetch_metric({MetricKey, [MetricType = {type, histogram} ]}) ->
     {MetricKey, rpc:call(?ERCHEF, folsom_metrics,get_histogram_statistics, [MetricKey]), MetricType};
```

Comment:
You'll want to change this one too, right? This is the one EC11 matches, and we want to make sure we give sane values there.